src/main/resources/db/migration/V2__adding_role_data.sql

```diff
@@ -4,3 +4,26 @@ insert into Role (roleName, location, capability, band, closingDate, status) val
 insert into Role (roleName, location, capability, band, closingDate, status) values
 ('roleNameTest2', 'locationTest2', 'capabilityTest2', 'bandTest2', CURRENT_DATE(), 'closed');
 
+INSERT INTO Role (roleName, location, capability, band, closingDate, status) values
```

Comment:
band and capability should be a separate table with foreign keys here as we will need the ability to create, edit and delete them in the future